dipy/segment/quickbundles.py

```diff
@@ -5,102 +5,110 @@
 from warnings import warn
 
 warn(DeprecationWarning("Module 'dipy.segment.quickbundles' is deprecated,"
-			" use module 'dipy.segment.clustering' instead"))
+                        " use module 'dipy.segment.clustering' instead"))
 
 
 class QuickBundles(object):
-    
-    def __init__(self,tracks,dist_thr=4.,pts=12):
-        """ Highly efficient trajectory clustering 
-        
+
+    def __init__(self, tracks, dist_thr=4., pts=12):
+        """ Highly efficient trajectory clustering
+
         Parameters
         -----------
         tracks : sequence of (N,3) ... (M,3) arrays,
                     trajectories (or tractography or streamlines)
-                    
-        dist_thr : float, 
+
+        dist_thr : float,
                     distance threshold in the space of the tracks
-        pts : int, 
-                number of points for simplifying the tracks 
-                       
+        pts : int,
+                number of points for simplifying the tracks
+
         Methods
         --------
         clustering() returns a dict holding with the clustering result
         virtuals() gives the virtuals (track centroids) of the clusters
-        exemplars() gives the exemplars (track medoids) of the clusters        
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
         Citation
         ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+
+        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
```

Comment:
Speaking of which -- any ideas on a strategy for removing support for this,
and expunging this code?

On Tue, Mar 1, 2016 at 8:41 AM, Marc-Alexandre Côté <
notifications@github.com> wrote:

> In dipy/segment/quickbundles.py
> <https://github.com/nipy/dipy/pull/953#discussion_r54594874>:
>
> >          Citation
> >          ---------
> > -
> > -        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
> > -
> > +
> > +        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
>
> I don't know, it's the old version which is deprecated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/953/files#r54594874>.
>